### PR TITLE
[CW-12361] fix: coin-tester cardano withdrawal bug

### PR DESCRIPTION
**Pull request by kanitsu**
----

*Checklist:*

- README.md includes:
  - [ ] The details of signing data and transaction data, this includes parameters.
  - [ ] Official docs or white paper.
  - [ ] Website or api can query assets and broadcast transaction.

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by cbx-tech** [comment]
Task linked: [CW-12361 [SDK] 測試 Cardano Staking - Withdraw 指令](https://app.clickup.com/t/25577573/CW-12361)

File: packages/coin-tester/src/components/coins/ada/txField.tsx [hunk 1]
```diff
@@ -150,7 +150,9 @@ function TxField(props: Props) {
       if (txType === TxTypes.Transfer) {
         const output = Number.parseInt(value[6]);
         if (!Number.isNaN(output)) change = diff - output;
-      } else if (keyDeposit && !Number.isNaN(keyDeposit)) {
+      }else if (txType === TxTypes.StakeWithdraw) {
```
Comment (aaronchou-coolbitx) [inline packages/coin-tester/src/components/coins/ada/txField.tsx:153]: 這邊 } 旁邊加上空格就可以 merge 了！

Comment (kanitsu) [reply packages/coin-tester/src/components/coins/ada/txField.tsx]: 已修改，謝謝。

**Review by hank121314 — APPROVED** [review]

File: packages/coin-tester/src/components/coins/ada/txField.tsx (hunk 1 above)
Comment (aaronchou-coolbitx) [reply packages/coin-tester/src/components/coins/ada/txField.tsx]: okay

**Review by aaronchou-coolbitx — APPROVED** [review]

**Pull request merged**